packages/tools/examples/segmentBidirectionalTool/index.ts

```diff
@@ -0,0 +1,465 @@
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  setVolumesForViewports,
+  volumeLoader,
+  getEnabledElement,
+} from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  addSliderToToolbar,
+  setCtTransferFunctionForVolumeActor,
+  addButtonToToolbar,
+} from '../../../../utils/demo/helpers';
+
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  SegmentationDisplayTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+  segmentation,
+  RectangleScissorsTool,
+  SphereScissorsTool,
+  CircleScissorsTool,
+  BrushTool,
+  PaintFillTool,
+  PanTool,
+  ZoomTool,
+  StackScrollTool,
+  StackScrollMouseWheelTool,
+  BidirectionalTool,
+  utilities: cstUtils,
+} = cornerstoneTools;
+
+const { roundNumber } = cstUtils;
+
+const { MouseBindings, KeyboardBindings } = csToolsEnums;
+const { ViewportType } = Enums;
+const { segmentation: segmentationUtils } = cstUtils;
+let renderingEngine;
+const viewportId = 'CT_AXIAL';
+
+// Define a unique id for the volume
+const volumeName = 'CT_VOLUME_ID'; // Id of the volume less loader prefix
+const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which defines which volume loader to use
+const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+const segmentationId = 'MY_SEGMENTATION_ID';
+const toolGroupId = 'MY_TOOLGROUP_ID';
+let representationUID;
+
+const actionConfiguration = {
+  contourBidirectional: {
+    method: cstUtils.segmentation.segmentContourAction,
+    bindings: [
+      {
+        key: 'c',
+      },
+    ],
+    segmentData: new Map(),
+  },
+};
+
+// ======== Set up page ======== //
+setTitleAndDescription(
+  'Segment bidirectional tool',
+  'Here we demonstrate automatic creation of the largest bidirectional tool which will fit the segment'
+);
+
+const size = '900px';
+const content = document.getElementById('content');
+const viewportGrid = document.createElement('div');
+
+viewportGrid.style.display = 'block';
+
+const element1 = document.createElement('div');
+element1.style.width = size;
+element1.style.height = size;
+
+// Disable right click context menu so we can have right click tools
+element1.oncontextmenu = (e) => e.preventDefault();
+
+viewportGrid.appendChild(element1);
+
+content.appendChild(viewportGrid);
+
+const instructions = document.createElement('p');
+instructions.innerText = `
+  Left Click: Use selected Segmentation Tool.
+  Use button to generate new bidirectional on largest slice.
+  Note that segment index 3 is consider to be 3,2 or 1 combined.
```

Comment:
move the explanation of segment index 3 to top, and in the drop down also say 3,2,1 instead of 3